Update choose-hybrid-identity-solution.md

martincoetzer opened the pull request:
Changed acronyms to PHS (Password Hash Sync). This article needs more updating to better frame PHS and PTA
#sign-off @jeffgilb

azurecla commented:
Hi __@ankitvijay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@tamram - please review contribution.

tamram commented:
#sign-off

martincoetzer committed:
Update choose-hybrid-identity-solution.md

Changed acronyms to PHS (Password Hash Sync). This article needs more updating to better frame PHS and PTA
